### ISSUE TITLE
after login the app stucks

### ISSUE THREAD

**Issue by juliusagodzo** [MahdiGhiasi/Xpotify#122]
<img width="901" alt="Cattura" src="https://user-images.githubusercontent.com/25937909/109430121-e7f2c300-79ff-11eb-8eeb-8880003fa3bc.PNG">

**Comment by juliusagodzo** [comment]
seems that the dev is no longer working on this...

**Comment by juliusagodzo** [comment]
@MahdiGhiasi

**Comment by Noobgamer0111** [comment]
Yeah, I logged in as well, but I don't think anything is happening.

**Comment by cutieshy** [comment]
could at least tell us its abandoned

**Comment by Noobgamer0111** [comment]
I think it's now abandonware. Oh well.

**Comment by mbs0ft** [comment]
rip the perfect spotify client?

**Comment by abg88** [comment]
Same here. And it looks abandoned for almost a year now. 
I wish there were a good soul who could a new fork of this.

**Comment by typicalGtaTG** [comment]
Although I'm not a 100% sure but looking at some of the code it seams like it has to do with "xpomusic.ghiasi.net" giving a 523 error which is what Xpo uses to log users in.
